bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/StartEventValidator.java

```diff
@@ -31,7 +33,18 @@ public Class<StartEvent> getElementType() {
   public void validate(
       final StartEvent element, final ValidationResultCollector validationResultCollector) {
     final Collection<EventDefinition> eventDefinitions = element.getEventDefinitions();
-    if (eventDefinitions.size() > 1) {
+
+    final Collection<EventDefinition> events =
+        eventDefinitions.stream()
+            .filter(
+                eventDefinition ->
+                    eventDefinition
+                        .getElementType()
+                        .getTypeName()
+                        .equals(BpmnModelConstants.BPMN_ELEMENT_SIGNAL_EVENT_DEFINITION))
+            .collect(Collectors.toList());
+
+    if (eventDefinitions.size() > 1 && eventDefinitions.size() > events.size()) {
```

Comment:
It's according the C7 docs [When deploying a process definition with one or more signal start events](https://docs.camunda.org/manual/7.17/reference/bpmn20/events/signal-events/#signal-start-event)

Reply:
No, I think that it's wrong. Please take a deeper look. 

The process can have multiple signal start events. But a single start event should refer only to one event definition. A signal start event should not refer to more than one signal event definition. Otherwise, the start event could be triggered by different signals.

Note that the `StartEventValidator` is invoked for a single start event.